src/stylesheets/blocks/_question.scss

```diff
@@ -0,0 +1,67 @@
+.question-section {
+  background-color: color(mint-green);
+  
+  @media (screen(lg)) {
+    margin-top: rem(-18);
```

Comment:
Ah, I can see you needed this to cover a gap between sections. I inspected and found the gap was being caused by a browser default bottom margin on the UL `tag-list`. If you set that to `margin: 0` that should take care of the gap.